projects/skps/configs/td-hm_hrnetv2-w18_skps-1xb16-160e_cofw-256x256.py

```diff
@@ -0,0 +1,176 @@
+custom_imports = dict(imports=['codecs2', 'models'])
```

Comment:
```suggestion
custom_imports = dict(imports=['custom_codecs', 'models'])
```
One more suggestion, `custom_codecs` would be more fridendly than `codecs2`

Reply:
Agree. Add to check list.